distribution/docs/src/main/jdocs/content/_security-framework/security-core-api-contents.adoc

```diff
@@ -0,0 +1,18 @@
+:title: Security Core API
+:type: subSecurityFramework
+:status: published
+:parent: Security Core
+:order: 00
+:summary: Security Core API.
+
+The Security Core API contains all of the ${branding} Security Framework APIs that are used to perform security operations within ${branding}.
```

Comment:
Reading " Security Framework APIs" I immediately wonder what is the Security Framework. Would it work to just says:
"The Security Core API contains all of the ${branding} APIs..."

Reply:
The intro to this section has a definition of the Security Framework, but I concede that your version is more concise and I don't think costs any information to be lost.